cmd/expiration-mailer/main_test.go

```diff
@@ -50,8 +51,7 @@ type fakeRegStore struct {
 func (f fakeRegStore) GetRegistration(ctx context.Context, id int64) (core.Registration, error) {
 	r, ok := f.RegByID[id]
 	if !ok {
-		msg := fmt.Sprintf("no such registration %d", id)
-		return r, core.NoSuchRegistrationError(msg)
+		return r, berrors.NotFoundError("no registration found")
```

Comment:
It looks like you dropped the `id` from the error text. It seems like that might be useful to have, is there a reason you dropped it?